src/base-config/keyboard.json

```diff
@@ -71,7 +71,13 @@
         }
     ],
     "edit.splitSelIntoLines": [
-        "Ctrl-Alt-L"
+        {
+           "key": "Ctrl-Alt-L"
```

Comment:
@abose I think we should not change the shortcut key without any strong reasons because users are accustomed to the shortcut keys in Brackets and changing any will not be received gladly by users.